src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```diff
@@ -70,6 +70,29 @@ public void setClientOperations(final Client client) {
     }
     commonSelector.wakeup();
   }
+  
+  @Override
+  public void startListening(final Server server) {
+    if(!checkServer(server)) {
+      return;
+    } else {
+      if(server.getServerType() == WireProtocol.TCP) {
```

Comment:
Yeah, that might be worth it, I was trying to avoid more specialized calls, but since UDP and TCP are so different its probably worth it.